src/features/page/simulation-list/SimulationList.tsx

```diff
@@ -57,9 +57,7 @@ export default function SimulationList(): JSX.Element {
   const [openSnackbar, setOpenSnackbar] = useState(false);
 
   useEffect(() => {
-    if (result.isError || result.isSuccess) {
-      setOpenSnackbar(true);
-    }
+    setOpenSnackbar(result.isError || result.isSuccess);
   }, [result]);
```

Comment:
ten warunek jest nieco inny niz przed zmiana => na pewno tego chcemy?

Reply:
Tak, tzn zeby zamykac snackbar jezeli stan resultu nie ma errora lub success do pokazania, inna sprawa ze i tak nie wiem jak zrobić zeby przez split second nie było errora po tym jak dialog jest otawrty i klikniesz symulacje nową

Reply:
```
useEffect(() => {
const func = () => setOpenSnackbar(result.isError || result.isSuccess);
const interval = setInterval(func, 1000);
return () => clearInterval(interval);
}, [result])
```
moze cos takiego?

Reply:
Nie naprawia problemu

Reply:
to olac, naprawi sie potem